cfme/infrastructure/cluster.py

```diff
@@ -307,9 +288,8 @@ def step(self, *args, **kwargs):
 
     def resetter(self):
         """Reset the view"""
-        self.view.toolbar.view_selector.select('Grid View')
-        self.view.paginator.check_all()
-        self.view.paginator.uncheck_all()
+        self.view.entities.paginator.check_all()
```

Comment:
**REQUIRED:** I've seen issues here where we fall over if there are no datastores, so we should perhaps make it a conditional on the fact that the paginator is displayed?